mdbook-pdf-outline: 0.1.4 -> 0.1.6

HollowMan6 opened the pull request:
Define `pname` separately for `mdbook-pdf-outline`: `mdbook_pdf_outline`
due to PyPI tarball name change. Reference:

- https://pypi.org/project/mdbook-pdf-outline/0.1.4/#files
- https://pypi.org/project/mdbook-pdf-outline/0.1.6/#files

Change maintainer from `nix-julia` to `hollowman6` as agreed.

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [X] x86_64-linux
  - [X] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [X] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/prs-ready-for-review/3032/5273

ShamrockLee commented on pkgs/by-name/md/mdbook-pdf-outline/package.nix:
```diff
@@ -5,13 +5,13 @@
 python3Packages.buildPythonApplication rec {
-  pname = "mdbook-pdf-outline";
-  version = "0.1.4";
+  pname = "mdbook_pdf_outline";
```
The change of package source is non-trivial (as `pname` here is also taken by `fetchPyPI` to form `src`), but there lacks explanation in the commit description and the PR description.

Could you elaborate the reason behind such change?

ShamrockLee replied:
@SuperSandro2000 Do you think we should change the `pname` styling of a Python package upon the upstream tarball name changes, or define the `pname` for `fetchPypi` separately?

SuperSandro2000 replied:
```suggestion
  pname = "mdbook-pdf-outline";
```

HollowMan6 replied:
Fixed

ShamrockLee commented on pkgs/by-name/md/mdbook-pdf-outline/package.nix:
```diff
@@ -26,7 +26,7 @@ python3Packages.buildPythonApplication rec {
     homepage = "https://github.com/HollowMan6/mdbook-pdf";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ nix-julia ];
+    maintainers = with maintainers; [ hollowman6 ];
```
Non-blocking:

I prefer splitting maintainer changes into its own commit for clarity, as such changes are related to the merge-bot permissions.

HollowMan6 replied:
Now this is a separate commit for this PR.

HollowMan6 committed:
mdbook-pdf-outline: 0.1.4 -> 0.1.6

Define `pname` separately for `mdbook-pdf-outline`: `mdbook_pdf_outline`
due to PyPI tarball name change. Reference:

- https://pypi.org/project/mdbook-pdf-outline/0.1.4/#files
- https://pypi.org/project/mdbook-pdf-outline/0.1.6/#files

Signed-off-by: Hollow Man <hollowman@opensuse.org>

HollowMan6 committed:
mdbook-pdf-outline: change maintainer

from `nix-julia` to `hollowman6` as agreed

Signed-off-by: Hollow Man <hollowman@opensuse.org>

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/prs-already-reviewed/2617/2311
